docs/source/notebooks/tutorial.ipynb

```diff
@@ -278,30 +272,156 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "It is possible to interact with the `DataFrame`; for instance, we can see what the \"attributes\" of the datasets are by printing the columns."
+    "## Catalog Entries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "3547"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "len(col.keys())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Intake ESM uses information saved in the `key_template` property in order to figure out how to create catalog entry keys:"
```

Comment:
Intake ESM uses the `key_template` property in order to create catalog entry keys